src/GirCore.sln

```diff
@@ -204,11 +205,11 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "GirTool", "Generation\GirTo
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "properties", "properties", "{BE2159DE-B264-4C71-9B52-94C5C956F76A}"
 	ProjectSection(SolutionItems) = preProject
-		properties\GirCore.Libraries.props = ..\properties\GirCore.Libraries.props
-		properties\GirCore.Logging.props = ..\properties\GirCore.Logging.props
-		properties\GirCore.Publishing.props = ..\properties\GirCore.Publishing.props
-		properties\GirCore.Testing.props = ..\properties\GirCore.Testing.props
-		properties\GirCore.Tooling.props = ..\properties\GirCore.Tooling.props
+		properties\GirCore.Libraries.props = properties\GirCore.Libraries.props
```

Comment:
Please restore the original paths. This breaks the files inside the solution folder "SolutionItems/properties/...".